docs/framework/pwgcf.md

```diff
@@ -3,14 +3,13 @@ sort: 5
 title: PWG-CF
 ---
 
-# Correlations and fluctiations (CF) analysis framework
+# Correlations and fluctuations (CF) analysis framework
 
 ## Get started!
 
-## Contact 
+## Contact
 
 ## Code
 
 - The codebase can be found in the
-[`PWG-CF`](https://github.com/AliceO2Group/O2Physics/tree/master/PWGCF) directory.
-.h).
+[`PWGCF`](https://github.com/AliceO2Group/O2Physics/tree/master/PWGCF) directory.
```

Comment:
The standard nomenclature since the beginning of ALICE is to hyphenate these (see https://alice-collaboration.web.cern.ch/organization/phb/index.html). Please keep hyphenated.

Reply:
But these lines are referring to the repository directories with their literal names, not the names of the PWGs.

Reply:
It's like if you sent a link to `https://aliceo2group.github.io/analysis-framework/docs/framework/pwghf.html` to someone but called the file `pwg-hf.html` in the test instead.

Reply:
Yes, I see your point, but I would still think it's better to refer to the PWG-CF _concept_ - which is hyphenated - in this case. It's true that in this sentence one could use both the _literal directory name_ or the _concept_ itself, but I would tend towards writing the correct hyphenated option as much as possible (and the hyperlink shows the right directory anyways).

Reply:
I understand that you want to use the *concept* name as much a possible and that's why I didn't change it anywhere else since I agree with you. But specifically here the context is clearly code-oriented. This is the Code section and it tells users what the code files are called and in which directories to find them. If things should be called by their proper names anywhere, it is right here. That's also why all the names in this section are rendered in the code font as opposed to the regular text font (see the HF and DQ pages).

One can refer to the directory implicitly with its *concept* name somewhere else in the text, like "the PWG-HF code needs more polishing" meaning *the code files used in the analyses of the PWG-HF group*, but that is a different context.

Reply:
Frankly, this is just not a relevant discussion to continue. Merging. Thanks!

Reply:
OK, thanks!